### PR TITLE
Added Algorithm for December 23

### PR DESCRIPTION
**Pull request by KrishnaAlagiri**
- Added Algorithm for December 23 and 24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -572,6 +574,54 @@ We have a small collection of algorithms, one for every day of the month. Scroll
       ```
 
 
+## **December 23 - Exam Seating Arrangement**
+  - **Problem**
+    - Exams happen all the time in our education system and a huge amount of effort is put into planning the seating arrangement to prevent students form malpractices. The task here is to create a system that does this automatically when the number of departments, the number of classrooms, and the students roll numbers are given and automatically generate the output plan.
```
Comment (akashsara) [inline README.md]: prevent students from* indulging in malpractice.

File: README.md [hunk 2]
```diff
@@ -572,6 +574,54 @@ We have a small collection of algorithms, one for every day of the month. Scroll
...
+    - There is only one condition when it comes to seating the students is that **No two students of the same department should sit together**
+
+
+  - **Example**
```
Comment (akashsara) [inline README.md]: So there's a lot of inputs to keep track of, I think we can lose the starting roll number requirement and just let it start from automatically. 

Also add a line stating how the notation works. It's obvious, but better to be safe.

File: README.md [hunk 3]
```diff
@@ -572,6 +574,54 @@ We have a small collection of algorithms, one for every day of the month. Scroll
...
+    - In simple words, Recursion means a function calling itself.
+    - Recursion makes breaking down and reducing complexity of a program much easier.
+    - Recursion has the power of performing of a finite set of instructions for infinite times without iterative control structures such as `for` and `while`.
+    - `Note:` Refrain from using string functions.
+  - **Example**
+    ```
+    Enter String: string STRING
+    Reversed String: GNIRTS gnirts
+    ```
+  - **Resources:**
```
Comment (akashsara) [inline README.md]: https://medium.freecodecamp.org/how-recursion-works-explained-with-flowcharts-and-a-video-de61f40cb7f9

https://stackoverflow.com/a/717839

https://www.geeksforgeeks.org/recursion/

File: README.md [hunk 4]
```diff
@@ -572,6 +574,54 @@ We have a small collection of algorithms, one for every day of the month. Scroll
...
+    CS9   EC9   CS10    EC10
+    EC12  CS12  EC11    CS11
+    CS13  EC13  CS14    EC14
+    EC8   ____  EC15    CS15
+    ```
+
+
+## **December 24 - Reverse a String Using Recursion**
+  - **Problem**
+    - In simple words, Recursion means a function calling itself.
```
Comment (akashsara) [inline README.md]: Recursion simply refers to a function which calls itself. Recursive functions have two parts - a base case or stopping scenario and something that calls the same function with modified parameters. Check out the resources section for some great explanations of how it works. 

Once you understand recursion, you will find yourself able to break down and reduce the complexity of a program much more easily. While they certainly don't replace loops, they are excellent in certain scenarios.

For this question, you will have to recursively reverse a string. First write a program to reverse a string normally. Then try to convert it into a recursive function.

**Comment by KrishnaAlagiri** [comment]
### All Issues Resolved.

**Pull request merged**